data/blog/bitcoin-grants-july-2023.mdx

```diff
@@ -0,0 +1,265 @@
+---
+title: "First Wave of Bitcoin Grants"
+date: '2023-07-14'
+tags: ['OpenSats', 'grants', 'bitcoin']
+draft: false
+authors: ['dergigi']
+images: ['/static/images/blog/04-bitcoin.jpg']
+summary: "OpenSats is funding over a dozen projects in the bitcoin & lightning ecosystem."
+---
+
+"The computer can be used as a tool to liberate and protect people, rather than
+to control them," as Hal Finney wrote so presciently 30 years ago.[^fn-hal]
+
+The goal of OpenSats is to help build the tools that Hal alluded to. Tools that
+liberate and protect, rather than systems that control and oppress. Many tools
+still have to be built. Many tools still need to be improved. However, "the
+universe smiles on encryption," as Assange so aptly put it.[^fn-assange]
+
+We believe that freedom tech is what carries this smile forward, which is why we
+are delighted to announce grants for over a dozen projects in the bitcoin &
+lightning ecosystem.
+
+[^fn-hal]: Hal Finney: [_Why remailers..._](https://cypherpunks.venona.com/date/1992/11/msg00108.html) (November 1992)
+
+[^fn-assange]: Julian Assange: [_A Call to Cryptographic Arms_](https://cryptome.org/2012/12/assange-crypto-arms.htm) (October 2012)
+
+The following open-source projects were [selected](https://github.com/OpenSats/website/blob/master/docs/grant-selection.md) to by the
+OpenSats board for funding:
+
+- [Payjoin Dev Kit](https://github.com/payjoin)
+- [Bolt12 for LND](https://github.com/ziggie1984)
+- [Splicing](https://lightningsplice.com/)
+- [Raspiblitz](https://github.com/raspiblitz/raspiblitz)
+- [Labelbase](https://github.com/Labelbase/Labelbase)
+- [BTCPay Server](https://github.com/btcpayserver)
+- [ZeroSync](https://github.com/zerosync/zerosync)
+- [Mutiny Wallet](https://github.com/MutinyWallet)
+- [next-auth Lightning Provider](https://github.com/jowo-io/next-auth-lightning-provider)
+- [Cashu](https://github.com/cashubtc/cashu)
+- [lnproxy](https://github.com/lnproxy/lnproxy)
+- [Blixt Wallet](https://github.com/hsjoberg/blixt-wallet)
+
+In addition to the open-source projects listed above, three
+educational initiatives were selected:
+
+- [Bitcoin Education in Nigeria](https://geyser.fund/project/bitcoineducationinnigeriawithapatajohnson)
+- [21 Ideas](https://www.21ideas.org/)
+- [CoreDev.tech](https://coredev.tech/)
+
+The first is an initiative started and led by Apata Johnson. Apata's project
+aims to educate youths on bitcoin and the opportunities it brings for the people
+living in the rural areas of Nigeria.
+
+The second, 21 Ideas, is a project that aims to
+bring quality Bitcoin education to Russian citizens. Tony and others have been
```

Comment:
Might be worthwhile to link to Apata and Tony's Twitter or Nostr profiles, I don't know who they are.